packages/appsync-modelgen-plugin/src/visitors/appsync-visitor.ts

```diff
@@ -271,7 +295,10 @@ export class AppSyncModelVisitor<
   protected queryMap: CodeGenQueryMap = {};
   protected mutationMap: CodeGenMutationMap = {};
   protected subscriptionMap: CodeGenSubscriptionMap = {};
-  protected typesToSkip: string[] = [];
+  protected inputObjectMap: CodeGenInputObjectMap = {};
+  protected unionMap: CodeGenUnionMap = {};
+  protected interfaceMap: CodeGenInterfaceMap = {};
+  protected typesToSkip: string[] = ['AMPLIFY'];
```

Comment:
Is it confirmed by the Studio team that we skip this input type?

Reply:
Yes it is not needed. The `AMPLIFY` is an input type and it is not generated either before or after.  I've also got the approval for the PR change from the studio team member.